Dockerfile

```diff
@@ -4,13 +4,17 @@ FROM golang:alpine as builder
 # queries required to connect to linked containers succeed.
 ENV GODEBUG netdns=cgo
 
+# pass tag, branch or commit during build
```

Comment:
nit: Please write a full sentence with capitalization and full stop at the end.

Reply:
:+1: